example_token_test.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020-2022 CYBERCRYPT
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package encryptonize_test
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/cyber-crypt-com/encryptonize-lib"
+)
+
+func ExampleEncryptonize_CreateToken() {
```

Comment:
There should probably be some more explanatory comments here to match the other examples.

Reply:
Fixed